### ISSUE TITLE
A MISSING FORM LABEL

### ISSUE THREAD

**Issue by Echika** [ushahidi/platform#4802]
The 'SEARCH POST BY KEYWORD' button has a missing form label.  If a form control does not have a properly associated text label, the purpose of that form control may not be presented to screen reader users. Form labels  should also provide visible descriptions and larger clickable targets for form controls.

**Comment by Echika** [comment]
Thanks
